packages/components/pages/f-checkout/src/services/addressService/tenants/ie.js

```diff
@@ -0,0 +1,42 @@
+function isFullPostCode (postcode) {
+    return postcode?.length >= 5;
```

Comment:
Should this be a regex match instead of a length check as `ab!_90` would pass this check?

Reply:
As mentioned below, we use this logic for every tenant